projects/ngx-prx-styleguide/src/lib/image/image-loader.component.spec.ts

```diff
@@ -4,7 +4,8 @@ import { MockHalService } from '../hal/mock/mock-hal.service';
 import { DebugElement } from '@angular/core';
 import { ImageLoaderComponent } from './image-loader.component';
 
-describe('ImageLoaderComponent', () => {
+// TODO: convert to integration test
```

Comment:
The component's `load` event seems not to fire in Jest at all. I'm guessing that the `load` event is equivalent to setting the `onload` property on an [`Image`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image).

The following test fails, which means maybe JSDOM just doesn't bother loading images since you'll never see them? If my guess above is true, maybe that makes this another good candidate for a test suite that runs in a "real" browser (like Phantom/Cypress)

```
it('loads images', done => {
  const docBody = document.getElementsByTagName('body')[0];
  var img =  document.createElement('img');
  img.onload = () => jest.fn(() => console.log('loaded'));
  img.setAttribute('src', 'http://fillmurray.com/10/10');
  img.setAttribute('height', '10');
  img.setAttribute('width', '10');
  docBody.appendChild(img);
  document.addEventListener('DOMContentLoaded', function(event) {
    expect(img.onload).toHaveBeenCalled()
    done();
  });
});
```